lib/python/treadmill_aws/cli/admin/aws.py

```diff
@@ -0,0 +1,104 @@
+import click
+import pprint
+import logging
```

Comment:
i am surprised it passes lint, as logging should be before click in import list...